includes/content-distribution/class-outgoing-post.php

```diff
@@ -211,6 +215,7 @@ public function get_payload( $status_on_create = 'draft' ) {
 			'status_on_create' => $status_on_create,
 			'post_data'        => [
 				'title'          => html_entity_decode( get_the_title( $this->post->ID ), ENT_QUOTES, get_bloginfo( 'charset' ) ),
+				'author'         => $post_author,
```

Comment:
`Outgoing_Author::get_wp_user_for_distribution` can return `WP_Error`. We shouldn't send that to the payload.